app/views/home/_signin_jobseekers.html.slim

```diff
@@ -1,7 +1,5 @@
 h2.govuk-heading-m = t(".title")
 
-p
-  = t(".description")
-  = govuk_link_to t(".signup_link"), new_jobseeker_registration_path, class: "govuk-link--no-visited-state"
+p == t(".description", link: govuk_link_to(t(".signup_link"), new_jobseeker_registration_path, class: "govuk-link--no-visited-state"))
```

Comment:
I like `_html` instead of `==` but I don't really mind

Reply:
Yes, I think it's better too. Changed it